defaults/main.yml

```diff
@@ -1 +1,20 @@
-# defaults file for localectl
+# defaults file for localectl.
+
+# Set your systems language designation.
+# Example
+#localectl_locale: en_US.UTF-8
```

Comment:
Why do we have defaults that are commented out? If this is meant for purely documentation purposes, then these values should be in `README.md` and not here. If they're actual defaults, then they should not be commented out here.

Reply:
I have removed them from defaults and updated the README.